styleguide/content/soul/colors.md

```diff
@@ -0,0 +1,92 @@
+We provide color definitions which can be imported and used as follows:
+
+```css
+@import '@mavenlink/design-system/src/styles/colors.css';
+.example {
+  color: var(--palette-primary-base);
+}
+```
+<h5 class="swatch-title">Black and Grays</h5>
+<div class="color-container grays">
+  <div class="swatch-grey-x-dark">
+    <span class="text">Black</span>
+  </div>
+  <div class="rgba">`rgba(0, 0, 0, .87)`</div>
+  <div class="hex">`#000000`</div>
+</div>
+<div class="color-container grays">
+  <div class="swatch-grey-dark">
+    <span class="text">Dk Gray</span>
+  </div>
+  <div class="rgba">`rgba(0, 0, 0, .87)`</div>
+  <div class="hex">`#000000`</div>
+</div>
+<div class="color-container grays">
+  <div class="rgba">`rgba(0, 0, 0, .54)`</div>
+  <div class="hex">`#757575`</div>
+</div>
+<div class="color-container grays">
+  <div class="swatch-grey-base">
+    <span class="text">Md Gray</span>
```

Comment:
Can we unify the label with its name? It's a bit weird to call it grey base but also medium grey. 

Also, it looks like we are diverging on spelling as well. Gray vs grey.